public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Is this really needed? And since we are here, how would you check if the application is behaving as expected?

Reply:
Thanks for your question. It helped me learn something I hadn't taken into consideration until now.

manifest.json is a file that contains metadata and provides information about the web app.
In this case, the manifest.json file or the short_name is not needed
but in a real-world scenario, it would be really useful because it will contain the name that the users will search to find the app.

I'm sorry but I don't know yet, how to check if the application is behaving as expected.
Generally speaking, my bet would be to learn writing tests and use a testing framework like Jest for example.